src/libraries/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -197,6 +197,9 @@
     <Compile Include="$(CommonPath)Interop\Windows\WinSock\Interop.WSAConnect.cs">
       <Link>Common\Interop\Windows\WinSock\Interop.WSAConnect.cs</Link>
     </Compile>
+    <Compile Include="..\..\Common\src\Interop\Windows\WinSock\Interop.WSADuplicateSocket.cs">
```

Comment:
Shouldn't here be something like $(CommonPath) instead of relative path?

Reply:
Good catch!